src/ucp/core/ucp_context.c

```diff
@@ -141,11 +141,13 @@ const char *ucp_object_versions[] = {
 };
 
 const char *ucp_extra_op_attr_flags_names[] = {
-    [UCP_OP_ATTR_INDEX(UCP_OP_ATTR_FLAG_NO_IMM_CMPL)]    = "no_imm_cmpl",
```

Comment:
for my understanding, why removing no_imm_cmpl and force_imm_cmpl?

Reply:
it wasn't supposed to be there in the first place, it was a workaround until this PR